scanners/ios/ios.go

```diff
@@ -2,14 +2,47 @@ package ios
 
 import "github.com/bitrise-io/bitrise-init/models"
 
+// Scheme is an Xcode project scheme or target
+type Scheme struct {
```

Comment:
These struct definitions are only used in `utility.go` so are shared between the macOS and iOS scanner. While I'm not a fan of shoveling everything into a file called `utility.go`, I think it would still be a better place than here in `ios.go`